test/utils/ViewTest.scala

```diff
@@ -88,23 +88,18 @@ trait ViewTest extends UnitTest with GuiceOneAppPerSuite {
   }
 
   def buttonCheck(text: String, selector: String, href: Option[String] = None)(implicit document: Document): Unit = {
+    s"have a $text button" which {
+      s"has the text '$text'" in {
+        document.select(selector).text() shouldBe text
+      }
 
-    if (href.isDefined) {
-      s"have a $text button" which {
-        s"has the text '$text'" in {
-          document.select(selector).text() shouldBe text
-        }
-        s"has a href to '${href.get}'" in {
-          document.select(selector).attr("href") shouldBe href.get
-        }
+      if (href.isDefined) {
```

Comment:
Just to double check, are you meant to be only checking the role of button if the href is defined? or should the href.isDefined only be around the href check?

Reply:
Good spot, should be outside the href check